x-pack/plugin/eql/qa/common/src/main/resources/data/sample-multi.data

```diff
@@ -0,0 +1,52 @@
+[
```

Comment:
Add some empty join key values (`null`) or missing values and add some test queries that use optional fields as join keys. `sample by ?os [any where true] [any where true]`